relay-dynamic-config/src/global.rs

```diff
@@ -45,11 +45,28 @@ impl GlobalConfig {
 #[serde(default, rename_all = "camelCase")]
 #[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
 pub struct Options {
-    // Example:
-    // ```rs
-    // #[serde(default, rename = "relay.some-option.name")]
-    // pub some_option: Vec<u32>,
-    // ```
+    /// list of platform names for which we allow using unsampled profiles for the purpose
+    /// of improving profile (function) metrics
```

Comment:
```suggestion
    /// List of platform names for which we allow using unsampled profiles for the purpose
    /// of improving profile (function) metrics.
```

(same for comments below)